examples/niscope/graph-measurement-betterproto.py

```diff
@@ -181,12 +186,18 @@ async def MeasureGrpcScope(scope_service: niscope_grpc.NiScopeStub, channel, vi)
     except KeyboardInterrupt:
         pass
 
-
+grpc_scope = ()
 async def main():
-    grpc_scope = await OpenGrpcScope(resource, server_address, server_port)
-    await ConfigureGrpcScope(*grpc_scope)
-    await MeasureGrpcScope(*grpc_scope)
-    await CloseGrpcScope(*grpc_scope)
+    try:
+        global grpc_scope
+        grpc_scope = await OpenGrpcScope(resource, server_address, server_port)
+        await ConfigureGrpcScope(*grpc_scope)
+        await MeasureGrpcScope(*grpc_scope)
+    except Exception as e:
```

Comment:
@ni-pkhare , can exception handling be similar to other examples?

Reply:
@doshirohan , other examples handle a couple of cases in exception handling know. Would it not be possible to do that in better proto?

Reply:
For the case when server is not running and we run the client, a WinError exception is thrown instead of gRPC exception. That's why I am catching any type of exception in general.

Reply:
Ok

Reply:
@ni-pkhare   For the case of not found and Unimplemented what is the behaviour?

Reply:
Can we catch WinError and gRPC error separately instead of generic Exception? To be frank, I don't have problem with catching generic exception either as long as we have good error messages reported for UNIMPLEMENTED and NOT_FOUND cases.

Reply:
@doshirohan , @kruthi-NI I have made changes now regarding UNIMPLEMENTED case and when server is not running. Could you pl have a look at it?